services/py-api/src/server/routes/verification_routes.py

```diff
@@ -0,0 +1,50 @@
+from typing import Any
+from fastapi import APIRouter, Response, Depends
+
+from src.database.db_manager import DB_MANAGER, PARTICIPANTS_COLLECTION, TEAMS_COLLECTION
+from src.database.repository.participants_repository import ParticipantsRepository
+from src.database.repository.teams_repository import TeamsRepository
+from src.database.transaction_manager import TransactionManager
+from src.server.handlers.verification_handlers import VerificationHandlers
+from src.service.hackathon_service import HackathonService
+from src.service.verification_service import VerificationService
+
+
+verification_router = APIRouter(prefix="/hackathon/participants/verify")
+
+
+def _p_repo(db_manager: DB_MANAGER) -> ParticipantsRepository:
+    return ParticipantsRepository(db_manager, PARTICIPANTS_COLLECTION)
+
+
+def _t_repo(db_manager: DB_MANAGER) -> TeamsRepository:
+    return TeamsRepository(db_manager, TEAMS_COLLECTION)
+
+
+def _tx_manager(db_manager: DB_MANAGER) -> TransactionManager:
+    return TransactionManager(db_manager)
+
+
+def _h_service(
+    p_repo: ParticipantsRepository = Depends(_p_repo),
+    t_repo: TeamsRepository = Depends(_t_repo),
+    tx_manager: TransactionManager = Depends(_tx_manager),
+) -> HackathonService:
+    return HackathonService(p_repo, t_repo, tx_manager)
+
+
+def _v_service(
+    h_service: HackathonService = Depends(_h_service),
+) -> VerificationService:
+    return VerificationService(h_service)
+
+
+def _handler(v_service: VerificationService = Depends(_v_service)) -> VerificationHandlers:
+    return VerificationHandlers(v_service)
+
+
+@verification_router.get("", status_code=200)
```

Comment:
Shouldn't this be a `PATCH` method? According to [RFC 7231](https://datatracker.ietf.org/doc/html/rfc7231#section-4.3.1) "GET"  is used to retrieve resources, and here we are Updating a resource.

According to the same memo [PUT](https://datatracker.ietf.org/doc/html/rfc7231#section-4.3.4) should be used to replace or create a resource if the resource does not exist. In other words an UPSERT operation. 

Here, we are not replacing a whole resource; we are changing some of its fields, or in other words, we are doing "Partial content updates" and [the PATCH METHOD](https://datatracker.ietf.org/doc/html/rfc5789#section-2) should be used as stated in the section for "PUT".